packages/authorization-process/test/getClientById.test.ts

```diff
@@ -30,12 +31,17 @@ describe("getClientById", async () => {
       generateId(),
       genericLogger
     );
-    expect(client).toMatchObject(mockClient);
+    expect(client).toEqual(mockClient);
   });
   it("should throw clientNotFound if the client with the specified Id doesn't exist", async () => {
-    const client = getMockClient();
+    await addOneClient(getMockClient());
```

Comment:
```suggestion
```
There's no need of adding

Reply:
Yes you are right, but for testing purposes I thought it makes more sense to have a client and not find it than to have none at all